pom.xml

```diff
@@ -166,4 +166,40 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    
+    <profiles>
+        <profile>
+            <id>container</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>0.34.1</version>
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>sling-launcher-container:${project.version}</name>
+                                    <alias>sling-launcher-container</alias>
+                                    <build>
+                                        <contextDir>../../..</contextDir>
```

Comment:
I think it would be good to set up automated builds for this Docker image eventually ( not in scope of this PR ). Looking at https://stackoverflow.com/questions/36853002/writing-dockerfile-for-dockerhub-automated-builds, it looks like the Docker build is basically invoking a top-level Dockerfile. Maybe we should move the 'launch' Dockerfile where the plug-in expects it and leave the top-level Dockerfile as an entry point for Dockerhub automated builds?